contrib/ruby/ext/trilogy-ruby/trilogy-ruby.h

```diff
@@ -9,13 +9,15 @@
 #define TRILOGY_FLAGS_CAST_BOOLEANS 2
 #define TRILOGY_FLAGS_LOCAL_TIMEZONE 4
 #define TRILOGY_FLAGS_FLATTEN_ROWS 8
+#define TRILOGY_FLAGS_CAST_SUM 16
```

Comment:
I'm not 100% certain this is limited to aggregate functions either. This is a bit verbose, but what do you think about something like:

```suggestion
#define TRILOGY_FLAGS_CAST_ALL_DECIMALS_TO_BIGDECIMAL 16
```